codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ViewServiceSourceGeneratorSuite.scala

```diff
@@ -202,13 +197,13 @@ class ViewServiceSourceGeneratorSuite extends munit.FunSuite {
          |    new MyServiceViewProvider(viewFactory, viewId)
          |
          |  override final def serviceDescriptor: Descriptors.ServiceDescriptor =
-         |    ServiceOuterClass.getDescriptor().findServiceByName("MyService")
+         |    MyServiceProto.javaDescriptor.findServiceByName("MyService")
```

Comment:
This is still wrong, but let's tackle that in a separate PR.
With ScalaPB the descriptors can be found in `*Proto` classes. In the sample it should use `CustomerViewProto` where I think `CustomerView` is derived from the file name `customer_view.proto`.